misc/check_metfile.py

```diff
@@ -0,0 +1,64 @@
+# Script dat van een metfile leest en dan checkt of de profielinformatie en de metingen genoeg elementen hebben.
```

Comment:
volgens mij kan dit bestand weg. 
Opmerkingen in dit bestand zijn deels te gebruiken op 'tools/check_metfile.py'